src/modules/products/productsWrapper/ProductsWrapper.js

```diff
@@ -223,6 +224,9 @@ class ProductsWrapper extends React.PureComponent {
 
         //return the default Zero results template
         if (numberOfProducts === 0 && unbxdCoreStatus === searchStatus.READY) {
+            if (typeof onZeroResults !== 'undefined') {
```

Comment:
Can we use `executeCallback` here. [ref](https://github.com/unbxd/react-search-JS-SDK/blob/master/src/modules/multilevelFacets/MultilevelFacetsContainer.js#L154)